### ISSUE TITLE
Hide Host button when already hosting on mainmenu.xml

### ISSUE THREAD

**Issue by kwadroke** [kwadroke/coldestgame#1]
Host button still shows up when already hosting. Should be replaced by Resume
This worked on SF.net version before moving button.

**Comment by kwadroke** [comment]
Code starts at Line 960.
Does not hide host button.

**Comment by kwadroke** [comment]
```
   // Show or hide the resume button
   if (gui[mainmenu]->visible)
   {
      GUI* resumebutton = gui[mainmenu]->GetWidget("resumebutton");
      GUI* hostbutton = gui[mainmenu]->GetWidget("hostbutton"); 	   
      if (netcode->Connected()) {
         resumebutton->visible = true;
         hostbutton->visible = false;
      } else {
         resumebutton->visible = false;
         hostbutton->visible = true;
      }
   }
```

Need to move resume button beside host button

**Comment by kwadroke** [comment]
Fixed in 17059b5
